contentcuration/contentcuration/serializers.py

```diff
@@ -448,7 +449,7 @@ def update(self, instance, validated_data):
         return instance
 
     def get_node_ancestors(self, node):
-        return node.get_ancestors().values_list('id', flat=True)
+        return list(node.get_ancestors().values_list('id', flat=True))
```

Comment:
This change was made because otherwise, the serializer will return a QuerySet that is not JSON Serializable. It seems there is code within DRF that handles this, but it is not triggered when we call the serializer ourselves directly.